.gitignore

```diff
@@ -1,120 +1,120 @@
-# Created by https://www.gitignore.io/api/flask
```

Comment:
The fact that all these files are showing up as changed makes me think you've changed the line endings to CRLF (windows specific) instead of just LF.

This could cause issues when running the project on Mac or Linux (which you will do later).

You can configure Git to convert them back to LF line endings on GitHub by setting `git config core.autocrlf input`.

Or `git config --global core.autocrlf false` to configure git to work like this all the time on your machine.